tests/functional/src/main/java/org/mule/functional/functional/FunctionalTestNotification.java

```diff
@@ -40,7 +40,15 @@
 
     static
     {
-        registerAction("event received", EVENT_RECEIVED);
+        //TODO(pablo.kraan): find a better way to manage this situation.
```

Comment:
I'll revert this and run all the test to check if this is really necessary. I will let you know, meanwhile I'm looking this PR.